### DISCUSSION TITLE
DFN Model

### DISCUSSION THREAD

**Discussion by SourabhJadhav2** [pybamm-team/PyBaMM#2429] [General]
### PyBaMM Version

22.8

### Python Version

3.9

### Describe the bug

This is not a issue related to PyBaMM. 
While developing DFN model, we use five governing equations - solid concentration, electrolyte concentration, solid potential, electrolyte potential, voltage equation.

While solving for solid concentration and electrolyte concentration, there is initial values for PDEs in parameter set.
But solving for solid potential and electrolyte potential, there is no initial values for ODEs in parameter set.
How you are doing in PyBaMM DFN model?
can you give some reference, please??

### Steps to Reproduce

None

**Comment by SourabhJadhav2** [comment]
Anyone have idea?

**Comment by brosaplanella** [comment]
The potential equations do not have a time derivative, and therefore do not need an initial condition.

**Comment by YannickNoelStephanKuhn** [comment]
The potential equations are _algebraic_ equations; think of them as constitutive relations if you're from a mechanics background.

If you want to understand their effects better, I recommend this article, which starts with a summary of the DFN and then derives the SPMe from it: https://arxiv.org/abs/1905.12553
I found it very helpful for my understanding, as the algebraic equations naturally substitute into the partial differential equations, showing their "constitutive" nature.

**Comment by SourabhJadhav2** [comment]
Yeah, got it. 
Solved. Thank you.
